README.md

```diff
@@ -98,6 +98,15 @@ KlaviyoClient.profiles.getProfileMetricsTimelineById({
     count: 10,
     sort: 'desc'
 });
+
+//Unset properties on a profile.
+KlaviyoClient.profiles.unsetProfileProperties({
+    profileId: 'myProfileId',
+    properties: [
+        'favoriteFood',
+        'likesOnions'
+    ]
+});
```

Comment:
This is probably super nitty but it could be good to have a non-camelCase property in that example array as well. I get that for javascript camelCase is the proper syntax but a customer could potentially see that and think they need to use camelCase since its javascript code when really it should match the way the property looks in Klaviyo.

Reply:
That's a good thought, properties can also have spaces e.g. Magento Store